docs: reformat the documentation (#396)
ansys/ansys-sphinx-theme: doc/source/examples/sphinx-autoapi.rst

.. _sphinx-autoapi:

> Sphinx autoapi

Reviewer: ```suggestion
Sphinx AutoAPI
```

==============

> This section describes ansys-sphinx-theme endpoints, their capabilities, and how
> to interact with them programmatically.

Reviewer: ```suggestion
This section describes how to use Python to interact programmatically with the Ansys Sphinx Theme.
```
I'm not sure this is what you want for the introduction. It seems like it should be about Sphinx AutoAPI--not our theme?
